.github/workflows/merge-demo.yml

```diff
@@ -87,5 +85,4 @@ jobs:
           overwrite: ${{ matrix.overwrite }}
           penetration_test: false
           parameters:
-            -p PROMOTE=ghcr.io/${{ github.repository }}/${{ matrix.name }}:${{ env.ZONE }}
```

Comment:
This is inconsistent with merge-main.yml which doesn't include the "ghcr.io/", which doesn't seem right.

Reply:
Inconsistent is bad! I'll look into it. :)

Reply:
Also, the demo workflow is stale. I'll likely replace it with the one from the QuickStart.

Reply:
@basilv The inconsistent code was already removed.  I'll handle demo separately, since this PR has the potential to cause an outage it shouldn't be overloaded.